bulldozer/update_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bulldozer
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/palantir/bulldozer/pull"
+	"github.com/palantir/bulldozer/pull/pulltest"
+)
+
+func TestShouldUpdatePR(t *testing.T) {
+	ctx := context.Background()
+	testMatrix := []struct {
+		blacklistable   bool
```

Comment:
I think renaming this to `blacklistEnabled` will make more sense (same thing for `whitelistable`).